locale/cs_CZ/locale.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale.xml
```

Comment:
the path... also it seems that the way how the paths are written in all external plugins (that are in their own repositories) differ...